### PR TITLE
Item logistics additions

### PR DESCRIPTION
**Pull request by ziadoz**
- Allow the boolean `assumeInfiniteInventory` to be set.
- Allow `onHandSafetyFactorQuantity` to be set.
- Allow `inventoryAvailabilityThreshold` values to be set.
- Moved all Pangaea constants into own class as they can be used in all other XML class (`Feed`, `Item` and `ItemLogistics`).

### CODE REVIEW

File: src/Item/ItemLogistics.php [hunk 1]
```diff
@@ -257,6 +110,75 @@ public function setShipNodeSupply($mdsfamId, $vendorStockId)
...
+     * @param $value
+     * @param $unit
+     * @throws PangaeaException
+     */
+    public function setOnHandSafetyFactorQuantity($value, $unit)
+    {
+        $unit = mb_strtoupper($unit);
+
+        if (mb_strlen($unit) === 0) {
+            throw new PangaeaException('Unit cost currency cannot be blank');
```
Comment (weshooper) [inline src/Item/ItemLogistics.php]: Message reads as though refers to something else?

File: src/Item/ItemLogistics.php [hunk 2]
```diff
@@ -320,16 +247,12 @@ public function render()
     <isHazmat>false</isHazmat>
     <!-- END: Required Dummy Values -->
     <!-- START: Dummy LIMO Values -->
```
Comment (weshooper) [inline src/Item/ItemLogistics.php]: Is comment still relevant?

Comment (ziadoz) [inline src/Item/ItemLogistics.php]: Probably not. Once I get everything merged I'll take out all the comments in a separate commit though.

File: src/Item/ItemLogistics.php (hunk 1 above)
Comment (ziadoz) [inline src/Item/ItemLogistics.php]: Yeah, you're right. I've changed the message.

**Comment by weshooper** [comment]
:+1:

**Pull request merged**